cjs/interface/event-target.js

```diff
@@ -1,12 +1,45 @@
 'use strict';
 // https://dom.spec.whatwg.org/#interface-eventtarget
 
-const EventTarget = (m => /* c8 ignore start */ m.__esModule ? m.default : m /* c8 ignore stop */)(require('@ungap/event-target'));
+const wm = new WeakMap();
```

Comment:
instead of extending Node or ungap EventTarget all features are now here.
In the end I think this is better as it allow for easier changes (one repo) and doesn't let the code exposed to eventual Node changes that doesn't support needed features anyway.